### PR TITLE
Backend: Added mockBee 

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
MockBee
I have added the MockBee Backend and dependencies in Package.json . So ignore other files. Focus only on the Backend , Server.js, Package.json (in src folder ) .

Files to check
src/Server.js
src/Package.json
PS - Ignore other files. .

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
I think it had difficult to do it. But you did it. It is great. Good job.

File: src/pages/login/Login.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    <p>
+                    <input type="checkbox" />
+                    <label> Remember me </label>
+                    </p>
+                    <p><a href="#">Forgot Password?</a></p>
+                </div>
+
+                <button class="primary-button error-button">login</button>
+
+                {/* <a href="/pages/signup-page.html">Create new account<i class="fa fa-arrow-right"></i>
```
Comment (santhoshpemmaka) [inline src/pages/login/Login.js:33]: If possible remove comment lines.

**Comment by santhoshpemmaka** [comment]
Suggestion: If possible follow one rule for naming file extension. I think your following some files using .js file and some files .jsx file. If possible changes in the next PR request. 

Apart from code looks good and clean structure 💯 . Good to merge 👍

**Review by santhoshpemmaka — APPROVED** [review]

File: src/pages/login/Login.js (hunk 1 above)
Comment (Shubhamkr2610) [reply src/pages/login/Login.js:33]: okay .. I will remove it.. and take care of it

**Pull request merged**